Release 0.6.2 - Adaptive phase filter documentation (#380)
ASFHyP3/hyp3-docs: docs/guides/insar_product_guide.md

{% endblock %}

{% block processing_options %}
### Processing Options and Optional Files

There are several options users can set when ordering InSAR On Demand products, including setting some processing parameters and selecting additional files to include in the output product package.

!!! tip "New: Adaptive Phase Filter parameter is now customizable!"

>     There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.

Author: ```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```

>     There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.

Reviewer: ```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently available in the API and Python SDK, and is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```

>     There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to both the SDK and Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.

Reviewer: ```suggestion
    There is now an option to adjust the adaptive phase filter parameter value when submitting On Demand InSAR jobs. This option is currently only available in the API, but is coming soon to Vertex! Refer to the [Adaptive Phase Filter section](#adaptive-phase-filter) for more information.
```
